src/store/index.ts

```diff
@@ -19,7 +33,19 @@ const state: any = {
 
 const persistState = createPersistedState({
     paths: ['user'],
-    fetchBeforeUse: true
+    fetchBeforeUse: true,
+    storage: {
+        getItem: key => {
+            try {
+                return ls.get(key)
+            } catch(err) {
```

Comment:
Add comments what we have done here and why?

Reply:
Comments Added.
Thank You Sir for reviewing my code.